### DISCUSSION TITLE
RequestParser Error

### DISCUSSION THREAD

**Discussion by kennedyi** [pallets/werkzeug#2382] [Q&A] [answered]
I am building a flask application that leverages RequestParser. Using POST, Everything works as expected with werkzeug 2.0.3 but if 2.1.0 or higher is installed it returns {"message": "The browser (or proxy) sent a request that this server could not understand."} or if flask debug is on {"message": "Failed to decode JSON object: None"}

Is this by design and if so how can I resolve. Thanks.

**Comment by davidism** [comment, marked as answer]
Follow the issues on the relevant extensions:

* https://github.com/flask-restful/flask-restful/issues/936
* https://github.com/python-restx/flask-restx/issues/422

Note that Flask-Restful doesn't seem to be maintained, so it's more likely that Flask-Restx will get the fix, although it doesn't seem to have been maintained recently either. There are plenty of other API extensions for Flask that are active, including:

* https://github.com/marshmallow-code/flask-smorest
* https://github.com/4Catalyzer/flask-resty/
* https://github.com/greyli/apiflask
* https://github.com/plangrid/flask-rebar/

**Reply by kennedyi** [reply]
Thank you. I will check it out.
